app/views/case_management/clients/index.html.erb

```diff
@@ -1,9 +1,11 @@
-<% content_for :page_title, t(".title") %>
+<% content_for :page_title, @page_title %>
```

Comment:
nice! now we can reuse with different titles! If we have a search or some other kind of client list, I could imagine us making use of this in those controllers too.